assets/components/forms/formHOCs/withArrow.jsx

```diff
@@ -0,0 +1,28 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import SvgArrowRightStraight from 'components/svgs/arrowRightStraight';
```

Comment:
this might be a great point to start using [imported styles](https://github.com/guardian/support-frontend/pull/1183) in this page and these components, which will makes the HOCs more 'magical' as we don't have to remember to import the CSS

Reply:
👍 will look at doing that in a future PR.